### PR TITLE
AT_05.07_004 | New item page has Input field for text data

### PR DESCRIPTION
**Pull request by NinaPrud**
[[https://trello.com/c/tUQ2VJBv](url)](https://trello.com/c/tUQ2VJBv/714-at0507004-new-item-page-has-input-field-for-text-data)

### CODE REVIEW

File: cypress/e2e/newItemInputField.cy.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 /// <reference types="cypress"/>
 
+import newItemNames from '../fixtures/newItemNames.json'
+
 const PORT = Cypress.env("local.port");
 
-describe('New Item', () => {
+describe('newItemInputField', () => {
 
     it('Verify New Item has input field', () => {
```
Comment (EkBlinova) [inline cypress/e2e/newItemInputField.cy.js:9]: TC name (it) should include AT number, for ex.:   AT_05.07_004 | Verify new item page has Input field for text data

**Comment by EkBlinova** [comment]
> [https://trello.com/c/tUQ2VJBv](url)

This link doesn't work. Please put a working link from Trello here.

**Comment by NinaPrud** [comment]
https://trello.com/c/tUQ2VJBv/714-at0507004-new-item-page-has-input-field-for-text-data
It is correct link. 

Thank you

**Review by lenashlyakhovets — APPROVED** [review]

**Review by EkBlinova — APPROVED** [review]

File: cypress/e2e/newItemInputField.cy.js (hunk 1 above)
Comment (NinaPrud) [reply cypress/e2e/newItemInputField.cy.js:9]: Yes, my TC name (it) includs number AT -  "AT_05.07_004 | New item page has Input field for text data", string 27

**Pull request merged**